cfme/tests/configure/test_access_control.py

```diff
@@ -37,6 +40,22 @@
 SETTINGS = VersionPicker({LOWEST: "Settings", "5.11": "User Settings"})
 
 
+FEATURES_510 = list(role_access_ui_510z["evmgroup-super_administrator"].keys()) + [
```

Comment:
Are these somehow specific to the test module here, or are these additional items in the access control tree that aren't included in cfme.roles?  If they're items that are included in the access control tree, then the data structures in cfme.roles should be updated, instead of just adding to them within this test module.

Reply:
Done, added in cfme.roles.py